packages/toolpad-core/package.json

```
@@ -29,14 +29,14 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mui/mui-toolpad.git",
+    "url": "https://github.com/mui/toolpad.git",
```

Wrong here

```suggestion
    "url": "git+https://github.com/mui/mui-toolpad.git",
```

https://publint.dev/@toolpad/core@0.6.0